src/Routing/Route/Route.php

```diff
@@ -651,7 +651,7 @@ public function match(array $url, array $context = [])
             isset($hostOptions['_port']) ||
             isset($hostOptions['_host'])
         ) {
-            $hostOptions += $context;
+            $hostOptions = (array)$hostOptions + $context;
```

Comment:
Why needed? Should always either be an array or throw exeption.

Reply:
`$hostOptions += $context;` is true

Reply:
this one gives weird error `Binary operation "+=" between (array&hasOffset('_host'))|(array&hasOffset('_port'))|(array&hasOffset('_scheme')) and array results in an error.`

probably phpstan issue, so i'm adding this to `ignoreErrors` instead